server/models/GuestToken.ts

```diff
@@ -0,0 +1,62 @@
+import { Model } from 'sequelize';
```

Comment:
Why are we adding a new Model here, as opposed to a table or something else?

Reply:
In the DB, it is a table (`GuestTokens`), the model is just it's translation to sequelize so we can easily use it in the code

Reply:
As for the reason to add it as a separate table rather than a column:
- Most profiles won't have a guest token. Having a guest token on verified organizations for example doesn't make much sense. When we do `models.Collective.findAll` it loads all fields by default, but 99% of the time we don't care about the guest token, so loading them is not useful.
- It's good to have some metadata about the tokens, like their creation date. With that in place we'll be able to expire them after a certain period if needed. Related benefit: it's easier to audit.
- `Collectives` table is already too big, and contains too much info. It's slow to edit. An alternative is sometimes to add the data to the `data` field, but querying it takes a long time (because it's JSON) and it's not typed, so it's easy to make mistakes when working with it.